apps/pie-storybook/stories/pie-button.stories.ts

```diff
@@ -190,9 +185,9 @@ const Template: TemplateFunction<ButtonProps> = ({
     responsiveSize,
 }) => html`
 <pie-button
-    size="${size}"
-    variant="${variant}"
-    type="${type}"
+    size="${size || defaultProps.size}"
```

Comment:
I think we should use `ifDefined` in all of these stories, like you've done for assistive-text above, sorry if I wasn't clear with my previous comment!

Reply:
Just to elaborate, because I won't be around to answer questions on Monday (if you have them).

In storybook, we want to be able to test that components apply default values properly when we don't pass in the prop. So in this example, if we use the storybook control to set `size` to `undefined`, `ifDefined` wouldn't pass in a value at all, and therefore the component has to decide which value to use, falling back to the default.